cli/src/commands/extensions/permissions.rs

```diff
@@ -261,6 +261,16 @@ impl From<&Permissions> for PermissionsOptions {
             allow_sys: None,
             allow_ffi: None,
             allow_hrtime: false,
+
```

Comment:
I don't think this whitespace is necessary. The long list of allow followed by the long list of deny should have a very clear "line" between them as long as they're not mixed together.